src/components/WebexInMeeting/WebexInMeeting.scss

```diff
@@ -16,6 +16,12 @@ $shadow-color: rgba(0, 0, 0, 0.2);
     }
   }
 
+  &.local-sharing {
+    .local-media-in-meeting {
+      top: 2.8rem;
```

Comment:
How does the transition look when switching between local view and share preview?